tomato_timer.py

```diff
@@ -94,7 +101,7 @@ def update_timer(self):
 
             # Bring back the window so user can easily start next tomato
             self.master.deiconify()
-            playsound('time_up.mp3', block=False)
+            playsound('C:\\Users\\tdsch\Documents\\Ubuntu-shared\\pomodoro\\time_up.mp3', block=False)
```

Comment:
Same here, can you change it for the relative path file? Also, now that we have two audio files, maybe we can a `resources/sounds` folder and place them there? What do you think?